ark/utils/segmentation_utils.py

```diff
@@ -1,10 +1,42 @@
 import os
 import copy
+import warnings
 
+import scipy.ndimage as nd
 import numpy as np
 import pandas as pd
 
-from ark.utils import io_utils
+import xarray as xr
+
+from skimage.feature import peak_local_max
+from skimage.measure import label, regionprops, regionprops_table
+
+import skimage.morphology as morph
+from skimage.segmentation import relabel_sequential
+import skimage.io as io
+
+from ark.utils import plot_utils, io_utils
+
+
+def modify_input_data(channel_data_xr, chan_list, fov):
+    """
+    Args:
+        channel_data_xr (xarray.DataArray):
+            xarray containing TIFs
+        chan_list (list):
+            list of channels
+        fov (int):
+            field of view
+
+    Returns:
+        input_data (numpy.ndarray):
+            ndarray containing data used for plotting
+    """
+    input_data = np.zeros(
+        (channel_data_xr.shape[1], channel_data_xr.shape[2], 3))
+    input_data[:, :, 1] = channel_data_xr.loc[fov, :, :, chan_list[0]].values
+    input_data[:, :, 2] = channel_data_xr.loc[fov, :, :, chan_list[1]].values
+    return input_data
```

Comment:
Can't we replace this function with `input_data = channel_data_xr.loc[fov, :, :, chan_list].values` and have `plot_overlay` deal with the number of channels/coloring logic @ngreenwald ?

Reply:
@ngreenwald

Reply:
Yes, we can have plot overlay handle this logic